### ISSUE TITLE
Icon tooltip

### ISSUE THREAD

**Issue by simakwm** [zamarrowski/react-ionicons#67]
All icons have a "title" prop with the name of the icon inside.
Is there a way to set this prop manually?

**Comment by zamarrowski** [comment]
I don't understand the answer because in the new version the title prox does not exists. Can you post a pic or a code with this issue?

**Comment by simakwm** [comment]
![Captura de Tela 2021-02-08 às 19 24 42](https://user-images.githubusercontent.com/665182/107289369-b069aa00-6a43-11eb-9838-ea6f438c1376.png)
![Captura de Tela 2021-02-08 às 19 54 00](https://user-images.githubusercontent.com/665182/107291808-80bca100-6a47-11eb-9679-50e7006890aa.png)
Here they are. Also, inspecting the code I found a <title></title> in every icon.

**Comment by zamarrowski** [comment]
Ouh! I didn't know this! But it's a good feature. Maybe tomorrow I will bump a new version with a new prop called title or tooltip. For example:

```js
<AirplaneOutline
  color={'#00000'} 
  height="250px"
  width="250px"
  title="Airplane"
/>
```

Or something like that.

Do you like these way to change the title?

**Comment by simakwm** [comment]
Sounds great if title defaults to "" =)

**Comment by zamarrowski** [comment]
I released a new version (4.2.0) adding the title prop. You can try this in the doc page changing the title prop 😄 
https://react-ionicons.netlify.app/

**Comment by simakwm** [comment]
I've just tested it in my project here. Sweet! 
Thank you!
